demo/gradio_fastcomposer.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import gc
+
+import gradio as gr
+import numpy as np
+import PIL
+import torch
+from mmengine import Config
+
+from mmagic.registry import MODELS
+from mmagic.utils import register_all_modules
+
+gc.collect()
+torch.cuda.empty_cache()
+register_all_modules()
+
+
+class ModelWrapper:
+
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
```

Comment:
Hi @xiaomile, when I run this gradio and inference demo by following th inscturction in readme.
I found this demo does not run with cuda by default.
These lines maybe need to added here.
```python
  self.device = torch.device('cuda' if torch.cuda.is_available(
      ) else 'cpu')
  self.model = self.model.to(self.device)
```